### PR TITLE
Elastic Agent: set FLEET_URL and FLEET_INSECURE

### PR DESCRIPTION
**Pull request by mtojek**
This PR sets the FLEET_URL to Kibana host. 

Either this or https://github.com/elastic/elastic-package/pull/272 will eventually work.

Issue: https://github.com/elastic/beats/issues/24310

### CODE REVIEW

File: internal/install/static_snapshot_yml.go [hunk 1]
```diff
@@ -91,6 +90,7 @@ services:
     - "FLEET_ENROLL=1"
     - "FLEET_ENROLL_INSECURE=1"
     - "FLEET_SETUP=1"
+    - "FLEET_URL=http://kibana:5601"
```
Comment (ruflin) [inline internal/install/static_snapshot_yml.go:93]: @blakerouse Is this required for the case where fleet-server runs as part of the Elastic Agent?

Comment (blakerouse) [reply internal/install/static_snapshot_yml.go:93]: With `FLEET_SERVER_ENABLE` set to true, `FLEET_URL` is ignored. `FLEET_URL` is required when `FLEET_SERVER_ENABLE` is not set.

**Review by ruflin — APPROVED** [review]
Tested with `elastic-package stack up --version=7.13.0-SNAPSHOT -v` and works as expected.

**Pull request merged**